src/commands/deployWorkspaceProject/getDefaultValues/getDefaultContextValues.ts

```diff
@@ -75,4 +73,6 @@ async function displaySettingsOverrideWarning(context: ISubscriptionActionContex
         { modal: true },
         { title: localize('continue', 'Continue') }
     );
+
+    ext.outputChannel.appendLog(localize('confirmedOverride', 'User confirmed deployment will target {0}"{1}" instead of existing workspace settings.', treeItemType, resourceName));
```

Comment:
Do you need a space between {0} "{1}"?

Reply:
No because of this section:

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/73a272ee-14e4-4cac-b69b-cb39ebd77630)

Reply:
lol that is so jank. Why do we do that??

Reply:
Mean Matt 🥲
![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/5290572/f11bc8ee-1adc-4935-8125-9ae75d86b311)

Reply:
Haha, because tree item could be invalid, but that's actually super edge case, so if you'd rather we just drop the else and have a normal spacing I'm fine with that too

Reply:
> Mean Matt 🥲 ![image](https://user-images.githubusercontent.com/5290572/277835826-f11bc8ee-1adc-4935-8125-9ae75d86b311.png)

I would avoid reading my edits because it takes me close to 9000 iterations before my writing is ready for human eyes 😂